test/built-ins/RegExp/prototype/flags/order.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2017 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 21.2.5.3
```

Comment:
we should use `esid`. We can use `es6id` as an extra but only if all the assertions in this file are valid for ES6. That's not the case.